faq.html

```diff
@@ -147,7 +147,7 @@ <h5><b>Q. What are the workshop, Developer Sprints and conference timings?</b></
           <h5><b>Q. What all does the Conference ticket (Oct 6 and Oct 7, 2018) include? </b></h5>
           <ul class="faq-answer">
                 <li>- Full conference pass for two days.</li>
-                <li>- Lunch and Hi-Tea</li>
+                <li>- Lunch and High-Tea</li>
```

Comment:
- Wouldn't access to Developer Sprints also be there. Also, what about workshops. We might need to provide complete information here. 
- We need this to sound and look attractive, Hence I suggest. 
A full conference pass for two days that includes access to all keynotes, talks, sessions, open-spaces and much more.